lib/observation_search.rb

```diff
@@ -29,20 +29,14 @@ def site_search_params(site, params = {})
 
     def search_in_batches(raw_params, options={}, &block)
       search_params = Observation.get_search_params(raw_params, options)
-      search_params.merge!({ page: 1, per_page: 500, preload: [ ],
+      search_params.merge!({ min_id: 1, limit: 500, preload: [ ],
         order_by: "id", order: "asc" })
-      scope = Observation.get_search_scope_or_elastic_results(search_params)
-      if scope.is_a?(ActiveRecord::Relation)
```

Comment:
Also, the AR:Relation find_in_batches will already to proper min_id/limit kinds of queries